drive.go

```diff
@@ -144,15 +145,15 @@ func (ds *DriveService) ListChildren(ctx context.Context, driveID string, filter
 	return items, resp, nil
 }
 
-// Search the hierarchy of items for items matching a query
-func (ds *DriveService) Search(ctx context.Context, q string, filter string) (*Items, *http.Response, error) {
-	q = url.QueryEscape(q)
-
-	if filter != "" {
-		q = q + "&filter=" + url.QueryEscape(filter)
+// Search the hierarchy of items for all items the user has access to.
+func (ds *DriveService) Search(ctx context.Context, q string) (*Items, *http.Response, error) {
+	// We need to escape single quotes with a double single
```

Comment:
I guess drop the `a` from this? E.g. `We need to escape single quotes with double single quotes, otherwise the request will fail with an unterminated error` ?